src/pybind/mgr/cephadm/utils.py

```diff
@@ -43,6 +43,13 @@ class ContainerInspectInfo(NamedTuple):
     repo_digests: Optional[List[str]]
 
 
+class SpecialHostLabels():
```

Comment:
I'm curious why this is not an enum.  Otherwise, +1 to using constants.

Reply:
made this class an enum/str so I didn't have to worry about type signatures for things like the `has_label` func that we've always passed strings before